packages/p2p-v2/src/components/FullPageMobileWrapper/FullPageMobileWrapper.tsx

```diff
@@ -4,13 +4,15 @@ import LeftArrowIcon from '../../public/ic-arrow-left.svg';
 import './FullPageMobileWrapper.scss';
 
 type TFullPageMobileWrapperProps = {
+    noFooter?: boolean;
     renderFooter?: () => React.ReactNode;
     renderHeader?: () => React.ReactNode;
     shouldFixedFooter?: boolean;
 };
 
 const FullPageMobileWrapper = ({
     children,
+    noFooter = false,
```

Comment:
We can just check if user provides `renderFooter`, if not by default there should be no footer